src/server/object_services/map_builder.js

```diff
@@ -116,7 +119,9 @@ class MapBuilder {
 
         /** @type {nb.Chunk[] } */
         const chunks_to_build = [];
-
+        if (this.purge_chunks) {
+            Array.prototype.push.apply(chunks_to_delete, live_chunks);
```

Comment:
We have a new syntax for that :)
```suggestion
            chunks_to_delete.push(...live_chunks);
```